### PR TITLE
Bump version to 1.5.4

### PR DESCRIPTION
**Pull request by nothingisfunny**
Bump to 1.5.4

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## v1.5.4 (2018-12-10)
+
+- Add ability to pass api key to individual API method calls
+
```
Comment (erdey) [inline Changelog.md]: do we have examples of this documented?

Comment (nothingisfunny) [reply Changelog.md]: @erdey The reason for adding this were issues that we faced when switching between multiple api keys within an internal service, but I think our customers would only use one api key at a time ever anyways.

Comment (erdey) [reply Changelog.md]: @nothingisfunny 👍 seems fine to leave it undocumented for now then

**Comment by nothingisfunny** [comment]
@jperichon Would you mind pushing to Ruby Gems again if everything looks fine to you? Thank you very much.

**Pull request merged**